packages/node_modules/@webex/react-component-activity-item-base/src/__snapshots__/index.test.js.snap

```diff
@@ -30,6 +30,7 @@ exports[`ActivityItemBase component renders properly 1`] = `
         className="webex-published published"
       >
         2016-09-20T19:52:57.186Z
+         
```

Comment:
**These below new lines are getting automatically by jest whenever we run test case.**